content/applications/hr/lunch/vendors.rst

```diff
@@ -0,0 +1,152 @@
+- :guilabel:`Location`: select which locations are able to order from this vendor. Multiple
+  locations can be selected. If this field is left blank, **all** locations can order from the
+  vendor.
+
+  .. note::
+     By default, an `HQ Office` location is created by default when creating a database, and is
+     available to select from the list.
+
+- :guilabel:`Send Order By`: click the radio button to select how orders are sent to the vendor. The
+  available options are :guilabel:`Phone` or :guilabel:`Email`.
+- :guilabel:`Order Time`: this field **only** appears if :guilabel:`Email` is selected for
+  :guilabel:`Send Order By`. Enter the time that an order must be emailed for the order to be
+  accepted. Enter the time in a `HH:MM` format, then select either :guilabel:`AM` or :guilabel:`PM`
+  from the drop-down next to the time field.
+
+.. image:: vendors/orders.png
+   :align: center
+   :alt: The orders section of a vendor form, with all fields filled out.
+
+.. _lunch/extras:
+
+Extras
+======
+
+When ordering an item in the *Lunch* app, optional extra items, sometimes referred to as *add-ons*,
+can be shown. These can be configured in any manner that suits the products being offered.
+
+By default, Odoo allows for three types of extra items, which can be thought of as *categories*. By
+default, the first type or *category* of add-ons is labeled `Extras`, the second is labeled
+`Beverages`, and the third is labeled `Extra Label 3`.
+
+.. important::
+   When configuring the extras, it is important to keep in mind that all the extras configured will
+   appear for **every item** offered by the vendor. That means that only items which apply to
+   **all** products from the vendor should be added.
+
+.. _lunch/configure-extras:
+
+Configure extras
+----------------
+
+Enter the following information for each of the three available extra sections:
+
+- :guilabel:`Extra (#) Label`: enter a name for the type of extra, such as `Toppings`. This can be
+  thought of as a *category*.
+- :guilabel:`Extra (#) Quantity`: select how the extras are selected. The options are:
+
+  - :guilabel:`None or More`: select this if the user is not required to make a selection.
+  - :guilabel:`One or More`: select this to **require** the user to make **at least one** selection.
+  - :guilabel:`Only One`: select this to **require** the user to **make only one** selection.
+
+Add extras
+----------
+
+After the labels and quantities have been configured for an extra category, the individual extra
+items must be added for each category.
+
+Click :guilabel:`Add a line` at the bottom of the list that appears on the right-hand side of the
+extra category. Enter the :guilabel:`Name` and :guilabel:`Price` for each item being added. The
+price can remain at `$0.00` if there is no cost. This is common for items like disposable silverware
+or condiments.
+
+.. example::
+   For a pizzeria that only offers personal pies, see their extras configured as follows:
```

Comment:
```suggestion
   For a pizzeria that only offers personal pizzas, see their extras configured as follows:
```
I fully understand what a "pie" means from a pizzeria...haha this suggestion is purely to ensure that nothing gets "lost in translation" with our global users...

Reply:
OMG I didn't think about that! One of those words I just take for granted ^_^